### ISSUE TITLE
Add SMS notification support for current list of mobile networks Email to SMS gateways.

### ISSUE THREAD

**Issue by chris001** [webmin/webmin#1188]
Webmin's list of email to SMS gateway addresses seem to be only a small fraction of the available gateways right now.

Here are seven updated addresses, which could/should all be added to webmin, to support nearly all current mobile providers.  Note, some are already in CSV or Excel format, making it very easy to import to Webmin:

1. https://kb.sandisk.com/app/answers/detail/a_id/17056/~/list-of-mobile-carrier-gateway-addresses
2. https://support.teamunify.com/en/articles/227
3. https://raw.githubusercontent.com/mfitzp/List_of_SMS_gateways/master/email2sms.csv
4. https://en.wikipedia.org/wiki/SMS_gateway
5. https://www.opentextingonline.com/emailtotext.aspx
6. https://smsemailgateway.com/
7. https://avtech.com/articles/138/list-of-email-to-sms-addresses/

**Comment by jcameron** [comment]
Thanks! I'll take a look at integrating these.

Is there one list that's the most complete?

**Comment by chris001** [comment]
> Thanks! I'll take a look at integrating these.
> 
> Is there one list that's the most complete?

It looks like each list seems to have some entries that other lists don't have!
Would you like me to combine them into one master CSV list that webmin could dynamically load from disk, and build the dropdown, when it's time to output the HTML page to the user?
What format would be needed, i.e., what column names: "Fill-in-format-email (ie +1xxxxxxxxxx@mobile.provider.net", "Country", "Name of Mobile Network"?

**Comment by jcameron** [comment]
Yes that would be really useful! And that format sounds fine to me..

**Comment by chris001** [comment]
OK, you can assign me to this issue!

**Comment by chris001** [comment]
I have the csv file of these phone numbers for texting if still needed.

**Comment by jcameron** [comment]
Yes, I'd be interested in seeing that CSV

**Comment by iliajie** [comment]
Up!
